pkg/vfs/utils_test.go

```diff
@@ -0,0 +1,43 @@
+package vfs
+
+import (
+	"io/fs"
+	"path/filepath"
+	"testing"
+	"testing/fstest"
+
+	qt "github.com/frankban/quicktest"
+)
+
+func TestFromDir(t *testing.T) {
```

Comment:
Nit: I would expect this test in `vfs_test.go` instead of `utils_test.go` as it is really testing the whole implementation with `fstest.TestFS`